modules/ztp-pre-staging-downloading-artifacts.adoc

```diff
@@ -0,0 +1,416 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-pre-staging-tool.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-downloading-images_{context}"]
+= Downloading the images
+
+The {factory-prestaging-tool} allows you to download the following images:
+
+* {product-title} images
+* Operator images that are included in the distributed unit (DU) profile for 5G RAN sites
+* Operator images from disconnected registries
+
+[NOTE]
+====
+The list of available Operator images can vary in different {product-title} releases.
+====
+
+[id="ztp-preparing-ocp-images_{context}"]
+== Preparing to download the {product-title} images
+
+To download the {product-title} container images, you need the current version of {rh-rhacm-first} is going to provision the SNO.
+The version of the hub cluster determines which Assisted Installer (AI) container images are used by the SNO to provision and report the inventory and progress of the spoke cluster.
+
+.Prerequisites
+
+* You partitioned the storage device.
+* You have enough space for the images on the partitioned device.
+* You conneced the bare-metal server to the Internet.
+* You have a valid pull secret.
+
+.Procedure
+
+. Check the {rh-rhacm} and Multicluster engine (MCE) version by running the following commands in the hub cluster:
++
+[source,terminal]
+----
+$ oc get csv -A | grep -i advanced-cluster-management
+----
+
+.Example output for {rh-rhacm} version
+[source,terminal]
+----
+open-cluster-management                            advanced-cluster-management.v2.5.4           Advanced Cluster Management for Kubernetes   2.5.4                 advanced-cluster-management.v2.5.3                Succeeded
+----
+
++
+[source,terminal]
+----
+$ oc get csv -A | grep -i multicluster-engine
+----
+
+.Example output for MCE version
+[source,terminal]
+----
+multicluster-engine                                cluster-group-upgrades-operator.v0.0.3       cluster-group-upgrades-operator              0.0.3                                                                   Pending
+multicluster-engine                                multicluster-engine.v2.0.4                   multicluster engine for Kubernetes           2.0.4                 multicluster-engine.v2.0.3                        Succeeded
+multicluster-engine                                openshift-gitops-operator.v1.5.7             Red Hat OpenShift GitOps                     1.5.7                 openshift-gitops-operator.v1.5.6-0.1664915551.p   Succeeded
+multicluster-engine                                openshift-pipelines-operator-rh.v1.6.4       Red Hat OpenShift Pipelines                  1.6.4                 openshift-pipelines-operator-rh.v1.6.3            Succeeded
+----
+
+//MCE is a prereq of ACM, so it's installed with ACM. If you install ACM 2.5, will MCE 2.0 install by default? Are these pairings "set" in this way?
```

Comment:
@alosadagrande?

Reply:
Yes, if you install ACM, then the proper MCE version will be installed as well. However, I think you can just install MCE standalone. Also as asked to RHACM folks, once you have your ACM/MCE operators running you can upgrade MCE to a different version than the one expected.

AFAIK there is no official matrix published, [here](https://coreos.slack.com/archives/CTDEY6EEA/p1666798259261459?thread_ts=1666779223.049819&cid=CTDEY6EEA) you can find more info.